src/universal/utils/objects.js

```diff
@@ -0,0 +1,12 @@
+
+/*
+ * A la:
+ * const stuff = { name: 'Thing', color: 'blue', age: 17 };
+ * const picked = pick(stuff, 'name', 'age');
+ */
+export function pick(o, ...fields) {
```

Comment:
`pick` is easy: `const {email, fullName} = invitee`
`without` is the one that get's messy, since there's no real ES2015 sugar. you could make a helper, but if i usually do something like `without(i, 'email')` as `const {email, ...inviteeWithoutEmail} = invitee`